src/gui/attributetable/qgsdualview.cpp

```diff
@@ -943,13 +954,45 @@ void QgsDualView::resizeColumn()
   }
 }
 
+void QgsDualView::resizeAllColumns()
+{
+  QAction *action = qobject_cast<QAction *>( sender() );
+  int col = action->data().toInt();
+  if ( col < 0 )
+    return;
+
+  QgsAttributeTableConfig config = mConfig;
+
+  bool ok = false;
+  int width = QInputDialog::getInt( this, tr( "Set column width" ), tr( "Enter column width" ),
```

Comment:
```suggestion
  int width = QInputDialog::getInt( this, tr( "Set Column Width" ), tr( "Enter column width" ),
```
HIG Guidelines are to use Title Case for dialog titles